script.js

```diff
@@ -0,0 +1,338 @@
+    theme3.style.background =  "hsl(177, 92%, 70%)"
+    //calculator header style change ends here
+    calculatorDisplay.style.background = "hsl(268, 71%, 12%)"
+    calculatorDisplay.style.color = "hsl(52, 100%, 62%)"
+    // calculator display style ends here
+    calculatorButtons.style.background = "hsl(268, 71%, 12%)"
+    operands.forEach(operand => {
+        operand.style.background = "hsl(268, 47%, 21%)"
+        operand.style.color = "hsl(52, 100%, 62%)"
+        operand.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    });
+    deleteBtn.style.background = "hsl(281, 89%, 26%)";
+    deleteBtn.style.color = "white" 
+    deleteBtn.style.boxShadow = "0px 4px 0px hsl(285, 91%, 52%)"
+    additionBtn.style.background = "hsl(268, 47%, 21%)"
+    additionBtn.style.color = "hsl(52, 100%, 62%)"
+    additionBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    subtractionBtn.style.background = "hsl(268, 47%, 21%)"
+    subtractionBtn.style.color = "hsl(52, 100%, 62%)"
+    subtractionBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    decimalPoint.style.background = "hsl(268, 47%, 21%)"
+    decimalPoint.style.color = "hsl(52, 100%, 62%)"
+    decimalPoint.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    divisionBtn.style.background = "hsl(268, 47%, 21%)"
+    divisionBtn.style.color = "hsl(52, 100%, 62%)"
+    divisionBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    multiplicationBtn.style.background = "hsl(268, 47%, 21%)"
+    multiplicationBtn.style.color = "hsl(52, 100%, 62%)"
+    multiplicationBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    allClear.style.background = "hsl(281, 89%, 26%)"
+    allClear.style.color = "white"
+    allClear.style.boxShadow = "0px 4px 0px hsl(285, 91%, 52%)"
+    equalityBtn.style.background = "hsl(176, 100%, 44%)"
+    equalityBtn.style.color = "hsl(198, 20%, 13%)"
+    equalityBtn.style.boxShadow = "0px 4px 0px hsl(177, 92%, 70%)"
+}
+// THE CALCULATOR CODE STARTS HERE
+let clearButton = document.querySelector(".allClear")
+let deleteButton = document.querySelector(".deleteBtn")
+// let calculatorDisplay = document.getElementById("calculatorDisplay");
+let secondDisplayContainer = document.querySelector(".secondDisplayContainer");
+let signs = document.querySelector(".signs");
+// let decimalPoint = document.querySelector(".decimalPoint")
+let NumberButton = document.querySelectorAll(".operands")
+let firstDisplay = document.querySelector(".first-display")
+let dummyDisplay = document.querySelector(".dummy-display")
+let add = document.querySelector(".additionBtn")
+let subtract = document.querySelector(".subtractionBtn")
+let multiply = document.querySelector(".multiplicationBtn")
+let divide = document.querySelector(".divisionBtn")
+let operationButton = document.querySelector(".equalsToBtn")
+let secondDisplay = document.createElement("div");
+NumberButton.forEach(button => {
+   button.addEventListener("click",eachnumber)
+    function eachnumber(){
```

Comment:
How about inlining the function (anonymously) in the addEventListener param?
```suggestion
   button.addEventListener("click",(){
```